itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/WarehouseInstance.java

```diff
@@ -75,15 +76,15 @@
 import static org.junit.Assert.fail;
 
 public class WarehouseInstance implements Closeable {
-  final String functionsRoot;
+  final String functionsRoot, repldDir;
```

Comment:
changing type in future for this will require significant work allover the code, unless there is immediate visibility. So just adding a new line for new type declaration will be the smallest of things to do if we decide to change type later.